lua/terrortown/menus/gamemode/guide/roles.lua

```diff
@@ -0,0 +1,120 @@
+--- @ignore
+
+CLGAMEMODESUBMENU.base = "base_gamemodesubmenu"
+
+CLGAMEMODESUBMENU.priority = 99
+CLGAMEMODESUBMENU.title = "submenu_guide_roles_title"
+
+local Key = Key
+local TryT = LANG.TryTranslation
+local GetPT = LANG.GetParamTranslation
+-- local sin = math.sin
+local cos = math.cos
+local abs = math.abs
+
+local mg = 10 -- margin
+local pd = 10 -- margin
+local iconSize = 112
+
+local specialCases = {
+	["traitor"] = function(params) return GetPT("info_popup_traitor_alone", params) end,
+	["executioner"] = function(params) return GetPT("ttt2_desc_executioner", params) end,
+	["sacrifice"] = function(params) return GetPT("ttt2_desc_sacrifice", params) end,
+}
+local teams = {
```

Comment:
I think you can get the actual list of role teams

Reply:
I tried looking for it, but wasn't able to, and they weren't sorted by number or anything I believe, so I just did it manually